packages/contentful--cra-template-create-contentful-app/template/src/index.tsx

```diff
@@ -31,6 +32,24 @@ if (process.env.NODE_ENV === 'development' && window.self === window.top) {
   render(<LocalhostWarning />, root);
 } else {
   init((sdk) => {
+    // You can create an instance of the the contentful-management client.
+    // This and the contentful-management package can be removed if its not used
+    // Visit https://www.contentful.com/developers/docs/extensibility/app-framework/sdk/#using-the-contentful-management-api
+    // to find out more
+    const cma = createClient(
+      { apiAdapter: sdk.cmaAdapter },
+      {
+        type: 'plain',
+        defaults: {
+          environmentId: sdk.ids.environment,
+          spaceId: sdk.ids.space,
+        },
+      }
+    );
+
+    // Use the plain or nested client without any additional configuration
```

Comment:
This feels a bit disconnected to me, and doesn't really explain itself.

I think change it to something like
```suggestion
    // You can use the cma client created above without any additional configuration
    // for example you can call cma.locale.getMany({}) to get a list of locales
```

Or just remove this entirely, as we have the link above explaining what the CMA client is for.